packages/tools/kadena-cli/src/tx/tests/createTransaction.test copy.ts

```diff
@@ -0,0 +1,21 @@
+import { describe, expect, it } from 'vitest';
+import { mockPrompts, runCommand } from '../../utils/test.util.js';
```

Comment:
fix filename? copy.ts?

Reply:
Oh huh, never even noticed that 🤣 thanks